mmtrack/core/bbox/iou_calculators/region_iou_calculator.py

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+try:
+    from vot.region import calculate_overlap
+except ImportError:
+    calculate_overlap = None
+
+
+def calculate_region_overlap(*args, **kwargs):
+    if calculate_overlap is None:
+        raise ImportError(
+            'Please run'
+            'pip install vot-toolkit@git+ssh://git@github.com/votchallenge/vot-toolkit-python@0c61b32'  # noqa: E501
```

Comment:
Note that, if we change to `pip install git+https://github.com/votchallenge/toolkit.git`, the `# noqa: E501` may be removed.